tests/acceptance/features/apiGraph/sharedWithMe.feature

```diff
@@ -0,0 +1,346 @@
+
+  https://owncloud.dev/libre-graph-api/#/me.drive/ListSharedWithMe
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario: user lists the file shared with them
+    Given user "Alice" has uploaded file with content "hello world" to "/textfile0.txt"
+    And user "Alice" has shared file "textfile0.txt" with user "Brian"
+    When user "Brian" lists the resources shared with him using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+{
+  "type": "object",
+  "required": [
+    "value"
+  ],
+  "properties": {
+    "value": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "name",
+          "id",
+          "remoteItem"
+        ],
+        "properties": {
+          "id": {
+            "type": "string",
+            "pattern": "^%share_id_pattern%$"
+          },
+          "name": {
+            "type": "string",
+            "enum": [
+              "textfile0.txt"
+            ]
+          },
+          "remoteItem": {
+            "type": "object",
+            "required": [
+              "createdBy",
+              "file",
+              "id",
+              "name",
+              "shared",
+              "size"
+            ],
+            "properties": {
+              "createdBy": {
+                "type": "object",
+                "required": [
+                  "user"
+                ],
+                "properties": {
+                  "user": {
+                    "type": "object",
+                    "required": [
+                      "id",
+                      "displayName"
+                    ],
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "pattern": "^%user_id_pattern%$"
+                      },
+                      "displayName": {
+                        "type": "string",
+                        "enum": [
+                          "Alice Hansen"
+                        ]
+                      }
+                    }
+                  }
+                }
+              },
+              "file": {
+                "type": "object",
+                "required": [
+                  "mimeType"
+                ],
+                "properties": {
+                  "mimeType": {
+                    "type": "string",
+                    "pattern": "text/plain"
+                  }
+                }
+              },
+              "id": {
+                "type": "string",
+                "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}\\$[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}![a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
I think there's a pattern checker for this one.
CC @grgprarup

Reply:
can you point out what is the pattern here like `^%share_id_pattern%$` this doesn't work here

Reply:
I think this one: https://github.com/owncloud/ocis/pull/7984/files

Reply:
that won't work for this case 😞

Reply:
I think the only difference is `!` and `$`, is that `drive-id` or `share-id`? If it is `drive-id`, we can add a new pattern in `substituteInLineCodes`.
In the case of share-id, it uses `:` as a separator. I think that might be an issue reported here https://github.com/owncloud/ocis/issues/8000

Reply:
what exactly is that id? if its a new pattern type of id then may be create new one.

Reply:
this is fileID, id of the file that is shared